docs/install/authentication.md

```diff
@@ -47,10 +47,19 @@ When you [install PLM from repositories](repos.md), the environment file is crea
 
 ### Example environment file 
 
-```{.text .no-copy}
-PLM_SOURCE_URI="mongodb://source:mys3cretpAssword@mysource1:27017,mysource2:27017,mysource3:27017/"
-PLM_TARGET_URI="mongodb://target:tops3cr3t@mytarget1:27017,mytarget2:27017,mytarget3:27017/"
-```
+=== "Replica sets"
+
+    ```{.text .no-copy}
+    PLM_SOURCE_URI="mongodb://source:mys3cretpAssword@mysource1:27017,mysource2:27017,mysource3:27017/"
+    PLM_TARGET_URI="mongodb://target:tops3cr3t@mytarget1:27017,mytarget2:27017,mytarget3:27017/"
+    ```
+
+=== "Sharded clusters"
```

Comment:
Please add some text explanation that for sharded clusters user needs to specify hostnames of mongos instances instead of mongod, so it's clear and user doesn't need to visually compare URI to find the difference